In [2]:
import pandas as pd
from scipy import stats
import numpy as np

# 1. State the Hypotheses statement:

W = 1000 + 5X;

Null Hypothesis(Ho):  u = W (no increase in weekly costs)

Alternative Hypothesis (H1): u > W (weekly costs have increased)

Because of the mean value is grater than meau value this is one-tailed test

# 2. Calculate the Test Statistic:

In [3]:
# Given Data
sample_mean = 3050
theoretical_mean = 4000 # Rs. (W = 1000 + 5X where X = 600)
std_dev = 125  #  (5 * 25)
sample_size = 25
alpha = 0.05 #(Critical Value)

In [4]:
# Calculating the test statistic (t)
t_statistic = (3050 - 4000) / (125 / np.sqrt(sample_size))
t_statistic

-38.0

# 3. Determine the Critical Value

In [6]:
# Determine the critical value from T-distribution for a one-tailed test
critical_value = stats.t.ppf(1 - alpha, df=sample_size - 1)
critical_value

1.7108820799094275

# 4. Make a Decision


In [11]:
# Compare the test statistic with the critical value and make a decision
if t_statistic > critical_value:   # for left-tailed test
  print("Reject the null hypothesis (Ho).")
else:
  print("Fail to reject the null hypothesis (Ho).")

Fail to reject the null hypothesis (Ho).


**Beacuse of we Fail to reject the null hypothesis(Ho) thats why we can concluded that there is insufficient evidence to conclude that weekly operating costs are higher than the model suggests.**




# 5. Conclusion:

Test Statistic (t): The test statistic is calculated as -38.0.

Critical Value: The critical value is 1.71.

Decision: Since the test statistic -38.0 is less than the critical value 1.71, we fail to reject the null hypothesis.

There we can conclude that there is insufficient evidence to support the restaurant owners' claim that the weekly operating costs are higher than what the theoretical model suggests.